<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPluginDialog</class>
 <widget class="QDialog" name="AddPluginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Plugin</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="noteLabel">
     <property name="text">
      <string>File paths and direct links are accepted.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="openButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="addPluginButton">
     <property name="text">
      <string>Add Plugin</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
